ci/docker/Dockerfile.build.arm64

```diff
@@ -27,6 +27,8 @@ ENV FC /usr/bin/${CROSS_TRIPLE}-gfortran
 ENV HOSTCC gcc
 ENV TARGET ARMV8
 
+RUN apt update && apt install -y unzip
```

Comment:
What is this required for? Could we move this part (as well as the OpenBLAS compilation) for all Dockerfiles into separate scripts so that they are centralized in the install folder?